api_spec.yaml

```diff
@@ -396,22 +414,19 @@ paths:
                      - status
                      - accessToken
                      - timeCreatedMs
+                     - queryString
                      - timeAccessTokenExpiresMs
                   - type: object
                     properties:
                       status:
                         type: string
                         enum: ['UNKNOWN_AUTH_CODE_ERROR',
-                          'AUTH_CODE_EXPIRY_ERROR', 
-                          'INVALID_REDIRECT_URI_ERROR',
+                          'AUTH_CODE_EXPIRY_ERROR',
                           'UNKNOWN_CLIENT_SECRET_ERROR', 
                           'UNKNOWN_CLIENT_ID_ERROR',
                           'REFRESH_TOKEN_EXPIRY_ERROR',
                           'UNKNOWN_REFRESH_TOKEN_ERROR',
-                          'UNKNOWN_SCOPE_ERROR',
-                          'INVALID_SCOPE_ERROR',
-                          'INVALID_QUERY_STRING_ERROR',
-                          'INVALID_CODE_VERIFIER_ERROR']
+                          'UNKNOWN_SCOPE_ERROR']
```

Comment:
Add status for SCOPE_NOT_ALLOWED_ERROR

Reply:
For when the input scope is not a sub set of the allowed scope

Reply:
changes made